content/projekte/_index.md

```diff
@@ -9,16 +9,88 @@ outputs:
 
 {{< headline-icon icon="icon-documents.svg" color="blue" >}}
 
-{{< headline-brackets-h1 color="red"  >}}
-Unsere Projekte
-{{< /headline-brackets-h1  >}}
+{{< headline-brackets-h1 color="red" >}}
+Projekte
+{{< /headline-brackets-h1 >}}
 
+{{< hero-block-center >}}
+[Alle Projekte](projekte/alle), die bei Code for Germany entstehen, sind Open Source. Sie lassen sich kostenlos verwenden und weiterentwicklen.
+{{< /hero-block-center >}}
 
-{{< hero-block-center  >}}
-In unseren Labs entstehen Projekte, die aufzeigen, welchen Mehrwert Offene Daten und Freie und Offene Software für unsere Gesellschaft bieten und wie diese Open Government begünstigen. 
-{{< /hero-block-center  >}}
+{{< paragraph-center >}}
+Seit der Gründung von Code for Germany im Jahr 2014 zeigen wir, was mit offenen Technologien alles möglich ist. Wir bauen Prototypen um selbst zu lernen; um zu zeigen, wie sich Technik zum Wohle aller einsetzen lässt; und um andere davon zu überzeugen, es uns gleich zu tun. Wir haben kein wirtschaftliches Interesse. Wir wünschen uns, dass unsere Projekte –  all unsere Apps, Plattformen, Sensoren, Prototypen, Ideen –  möglichst viel weiterverwendet werden und möglichst vielen Menschen nutzen.
 
+Du kannst dir gleich [alle Projekte](/projekte/alle) ansehen oder weiterlesen und erfahren, wie unsere Arbeit der Gesellschaft hilft.
+{{</ paragraph-center >}}
 
-{{< paragraph-center  >}}
-In unseren prototypischen Anwendungen werden die Offenen Daten z. B. durch Visualisierungen für alle Menschen zugänglich gemacht. So sind komplexe Informationen leichter zu verstehen. Die Projekte orientieren sich an den jeweiligen lokalen Bedarfen. Gleichzeitig sind sie darauf ausgerichtet, Synergien zwischen kommunalen Verwaltungen deutschlandweit zu schaffen.
-{{< /paragraph-center  >}}
+
+{{< projekte/featured-projects >}}
+
+{{< headline-brackets-h1 color="blue"  >}}
+Konkrete Fragen, die sich mit Offenen Daten beantworten lassen
+{{< /headline-brackets-h1 >}}
+
+{{< paragraph-center >}}
+
+Offene Daten und die Anwendungen, die daraus entsehen, bieten einen
+direkten Mehrwert in unserem Alltag. Sie machen Informationen
+übersichtlich und helfen uns, informierte Entscheidungen zu treffen.  
+
+### Auf welche Schule soll ich mein Kind schicken?
+
+Aus gesammelten Informationen und visualisierten Karten
+in [Leipzig](https://opengovtech.de/cfg-ziele/projekte/le-kitas_und_schulen_in_leipzig/), [Köln](https://opengovtech.de/cfg-ziele/projekte/cologne-schulen-in-koeln/),  [Berlin](https://opengovtech.de/cfg-ziele/projekte/be-berliner-schulen/),
```

Comment:
Warum beginnen die Links alle auf https://opengovtech.de/ ?

Reply:
```suggestion
in [Leipzig](/projekte/le-kitas_und_schulen_in_leipzig/), [Köln](/projekte/cologne-schulen-in-koeln/),  [Berlin](projekte/be-berliner-schulen/),
```

Reply:
Danke @joergreichert da hast du was gutes gefunden!

Reply:
@k-nut die ollen opengovtech.de Links sind überall im Text, auch in der Box zu Kommunalpolitik versteckt sich einer. Ich / Nora waren da offensichtlich nicht sauber und dann betriebsblind. 

Ich checke nicht, wie ich jetzt an dieser Stelle noch alle rauskommentieren kann. Entweder du fischst die alle raus wenn du die Änderungen einbaust, oder lass es einfach sein und ich gehe dann nochmal drüber beim nächsten Review. 

Sorry and thank you v much.